src/main/webapp/app/overview/course-overview.component.html

```diff
@@ -67,11 +67,16 @@
                                     }
                                 </li>
                             }
+                            <!-- More icon and label in case of window minimization -->
+                            <div [hidden]="!anyItemHidden" class="three-dots nav-link px-3" (click)="toggleDropdown()">
```

Comment:
Could this be (wrapped in) a `<li class="nav-item">` like the other items of the list?